rfc/disallow-inline-args/RFC.md

```
@@ -0,0 +1,669 @@
+
+```
+Parse (687) -> NormalizeOperation (830) -> NormalizeVariables (855)
+  -> RemapVariables (906) -> ValidateQueryComplexity (1038)
+  -> Validate (1059) -> ValidateStaticCost (1149) -> Plan (1119)
+```
+
+Detection is a single scan of `o.kit.doc.Arguments` run **immediately after `Parse()` (line 687)
+and before `NormalizeOperation()` (line 830)**. At every later line in the order above, normalization
+has already begun rewriting or pruning the operation, so just-after-parse is the only complete view.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)